og.og_permissions.yml

```diff
@@ -0,0 +1,14 @@
+update group:
+  title: 'Edit group'
+  description: 'Edit the group. Note: This permission controls only node entity type groups.'
+  'default role':
+    - OG_ADMINISTRATOR_ROLE
```

Comment:
We could make these strings 'special'. A bit like VERSION is now for info
yml files. Does this even work in it's current form? Doesn't the YAML
parser just interpret that as a string?
On 29 Nov 2015 17:45, "Amitai Burstein" <notifications@github.com> wrote:

> In og.og_permissions.yml
> <https://github.com/amitaibu/og/pull/47#discussion_r46095435>:
>
> > @@ -0,0 +1,14 @@
> > +update group:
> > +  title: 'Edit group'
> > +  description: 'Edit the group. Note: This permission controls only node entity type groups.'
> > +  'default role':
> > +    - OG_ADMINISTRATOR_ROLE
>
> I'll open a PR for this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/amitaibu/og/pull/47/files#r46095435>.
>